atoMEC/staticKS.py

```diff
@@ -420,10 +420,14 @@ def calc_v_ha(density, xgrid):
         Notes
         -----
         v_ha is defined on the r-grid as:
-        .. math:: v_{ha}(r) = 4\pi*\int_0^r dr' n(r') \frac{r'^2}{\max(r,r')}
+
+        .. math:: 
+            v_\mathrm{ha}(r) = 4\pi*\int_0^r \mathrm{d}r' r'^2 \frac{n(r')}{\max(r,r')}
```

Comment:
```suggestion
            v_\mathrm{ha}(r) = 4\pi\int_0^r \mathrm{d}r' r'^2 \frac{n(r')}{\max(r,r')}
```
If `*` shall denote ordinary multiplication, leave it out.